sequencer/onchain.go

```diff
@@ -97,6 +97,13 @@ func (s *Sequencer) processTransitionOnChain() {
 			return true // Continue to next process ID
 		}
 
+		// If current contracts does not support blob transactions, set sidecar
+		// to nil to avoid sending it
+		if !s.contracts.SupportBlobTxs() {
+			log.Debug("blob transactions not supported by current contracts, skipping blob sidecar")
```

Comment:
No need for this debug message. This generates noise in the logs and do not add value. Actually, this check is unnecessary. There is no need to set the blobSidecar to nil, just sent the corresponding transaction on pushTransitionToContract()

Reply:
The tx is the same, the way that it is recovered is different. I moved the check to `pushTransitionToContract` but the flag is the same, a tx has not blobs if `blobSidecar` is nil.